src/annotator/sidebar.js

```diff
@@ -53,43 +49,52 @@ function createSidebarIframe(config) {
  * as well as the adjacent controls.
  */
 export default class Sidebar extends Guest {
+  /**
+   * @param {HTMLElement} element
+   * @param {Record<string, any>} config
+   */
   constructor(element, config) {
-    if (config.theme === 'clean' || config.externalContainerSelector) {
-      delete config.pluginClasses.BucketBar;
-    }
-
-    let externalContainer = null;
-
-    if (config.externalContainerSelector) {
-      externalContainer = document.querySelector(
-        config.externalContainerSelector
-      );
-    }
+    super(element, config);
 
     let externalFrame;
     let frame;
+    let hypothesisSidebar;
```

Comment:
Can you add a comment here that this refers to the `<hypothesis-sidebar>` element?

Once we've confirmed that the Shadow DOM approach is working then we can look to simplify some of the inner DOM structure and class names.

Reply:
Added comment and issue to track follow up actions (#2908)